mobile/mobile-core/android-process-service.ts

```diff
@@ -75,21 +77,65 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 				// Process information will look like this (without the columns names):
 				// USER     PID   PPID  VSIZE   RSS   WCHAN    PC         NAME
 				// u0_a63   25512 1334  1519560 96040 ffffffff f76a8f75 S com.telerik.appbuildertabstest
-				let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", "-a", "--text", processId]).wait();
-
-				return _.last(processIdInformation.trim().split(/[ \t]/));
+				let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", processId]).wait();
+				let appIdentifier = _.last(processIdInformation.trim().split(/[ \t]/));
```

Comment:
It appears this `split` splits by whitespace, followed by a tab - is this the desired effect, rather than splitting by whatespace/tab only?